src/driver/tests/experimental_design_test.py

```diff
@@ -190,6 +228,118 @@ def test_evaluate_with_two_cores(self):
             results = exp.load()
             self.assertEqual(results.shape[0], 8)
 
+    def test_evaluate_with_apache_beam_locally(self):
+        with TemporaryDirectory() as d:
+            data_design_dir = join(d, "data_design")
+            experiment_dir = join(d, "experiments")
+
+            results = self._evaluate_with_apache_beam(data_design_dir, experiment_dir)
+            # We don't check each column in the resulting dataframe, because these have
+            # been checked by the preceding unit tests.  However, we make a few strategic
+            # probes.
+            self.assertEqual(results.shape[0], 1)
+            self.assertAlmostEqual(results["relative_error_at_100"][0], 0.0, delta=0.01)
+            self.assertGreater(results["max_nonzero_frequency_from_halo"][0], 0)
+            self.assertEqual(results["max_nonzero_frequency_from_data"][0], 1)
+
+    @patch.object(experimental_design, "GSClient", LocalGSClient)
+    @patch.object(experimental_design, "EvaluateTrialDoFn", FakeEvaluateTrialDoFn)
+    @patch.object(data_set, "GSPath", LocalGSPath)
+    @patch.object(data_design, "GSPath", LocalGSPath)
+    def test_evaluate_with_apache_beam_with_cloud_path(self):
```

Comment:
Please turn off all of the log messages that this function is generating to stdout.